### ISSUE TITLE
mnemonic.txt file not recognized ("Uh-oh! No Keys Found")

### ISSUE THREAD

**Issue by dipique** [guydavis/machinaris#76]
This has to be user error, but I'm at a loss. I'm getting the "Uh-oh! No Keys Found" message when I open my container. "No valid keys were found at these path(s) in-container: `/root/.chia/mnemonic.txt`". The path looks right; the mnemonic setting for the container is set to `/root/.chia/mnemonic.txt`.

I copied my mnemonic.txt file  from the unraid host console using this command:

`root@Unraid:/mnt/user/chia# cp mnemonic.txt /mnt/user/appdata/machinaris/mnemonic.txt`

As a note, this `mnemonic.txt` file was used successfully with the official chia docker.

I verified that it was copied from inside the container console:

```
# cd /root/.chia
# ls
chiadog  machinaris  mainnet  mnemonic.txt  plotman  root # <-- removed asterisks, they were just formatting
```

Then I verified that the file actually contains the menomic using the command `cat mnemonic.txt` (I won't show the results for obvious reasons).

Finally, I restarted machinaris. But I still have the same error.

Please tell me I'm missing something silly.

**Comment by guydavis** [comment]
Sorry to hear this is happening.  That all looks right above, with possible exception of the `root` folder being inside the `/root/.chia` folder.    Also, the double asterisks around mnemonic.txt are a bit sus.

Let's see what the [logs](https://github.com/guydavis/machinaris/wiki/Troubleshooting#nothing-works--where-are-the-logs) show.  Please attach the contents of `/mnt/user/appdata/machinaris/machinaris/logs/webui.log`.  Also please take a look at the Machinaris container log as seen from Unraid's Docker tab, click on Machinaris, choose Logs.  

Please double-check the Variable for 'keys' in the Machinaris Docker Edit screen of Unraid.  Also, let's double-check file permissions with an `ls -al` like this:
![image](https://user-images.githubusercontent.com/12936/120422712-16957800-c326-11eb-9527-eccb95f175d4.png)

Hope the helps!

**Comment by dipique** [comment]
This morning, I started up the machinaris docker... and it came up just fine. I'm at a complete loss. Nothing happened between then and now, but it's fixed.

The oddities you noticed were artifacts of my testing. The asterisks were my attempt to make something **bold**, and the "root" folder instead `/root/.chia` was actually a file; a copy of `mnemonic.txt` I created when misusing the `cp` command.

I restarted the docker container a couple times, just to verify. Works like a charm. It's recognizing the plots and generally doing its thing.

Thanks for your help, and sorry there wasn't a more satisfying conclusion. I'll let you know if I see this again and will come prepared with logs and permission info.
